operations/patch.go

```diff
@@ -148,6 +148,20 @@ func Patch() cli.Command {
 				return errors.Errorf("can't define tasks/variants when reusing previous patch's tasks and variants")
 			}
 
+			if params.Alias != "" {
+				for _, alias := range conf.LocalAliases {
+					if alias.Alias == params.Alias {
```

Comment:
We should validate the alias at this point; if both variant and task both aren't populated, we should error. I'm also thinking we can support tags if we add these to the regular Tasks/Variants logic.

Reply:
done